docs/en/multi_modal/llava.md

```diff
@@ -6,8 +6,8 @@ LMDeploy supports the following llava series of models, which are detailed in th
 | :----------------------------------: | :--: | :------------------------: |
 | llava-hf/Llava-interleave-qwen-7b-hf |  7B  |     TurboMind, PyTorch     |
 |       llava-hf/llava-1.5-7b-hf       |  7B  |     TurboMind, PyTorch     |
```

Comment:
[llava-next](https://huggingface.co/collections/llava-hf/llava-next-65f75c4afac77fd37dbbe6cf) models like `llava-hf/llava-v1.6-mistral-7b-hf`  and `llava-hf/llava-v1.6-vicuna-7b-hf` are supported by pytorch yet not supported in turbomind, should we update it in the document?